src/client/common/process/condaService.ts

```diff
@@ -22,6 +24,7 @@ export class CondaService {
         @inject(IMemento) @named(GLOBAL_MEMENTO) private readonly globalState: Memento,
         @inject(IFileSystem) private readonly fs: IFileSystem
     ) {}
+    @traceDecorators.verbose('getCondaVersion', TraceOptions.BeforeCall)
```

Comment:
Add a bunch of tracing to track down slow downs. Figured it would be good to have these still. They simply emit something like:

```
getCondaVersion starting ...
...
...
getCondaVersion completed in xxx ms with truty result
```
So two extra lines to a log.